### PR TITLE
README file added with details

### PR DESCRIPTION
**Pull request by krrohitch**
README.md added with details. Do review.
Took a while to get acquainted with Markdown format. Hence, the multiple commits.

### CODE REVIEW

**Review by itaditya — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Website for Nibble Computer Society
+
+Nibble Computer Society is the official computer society of JSS Noida. We are a group of students working towards enhancing knowledge in the field of computer science. The website shows our contribution to the college, our members and events & workshops that we organize.
+
+  Visit here : [hackncs.com](http://hackncs.com)
+
+  Follow us here :
+- [Facebook](https://www.facebook.com/nibblecomputersociety)
+- [Behance](https://www.behance.net/ncs-jss)
```
Comment (ankitjain28may) [inline README.md:9]: This link is not opening.. Either fix it or remove it.

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

**Comment by ankitjain28may** [comment]
@krrohitch Squash your commits as well.

**Comment by itaditya** [comment]
Let's leave the squashing part, what say Ankit??

**Comment by ankitjain28may** [comment]
6 +1 commits are more enough in one PR so i think its better to squash them.

**Comment by shlok007** [comment]
@krrohitch this is fine for now but in future make sure to create a separate branch with your changes and avoid using your master at all times. This way you can easily work on multiple issues/changes simultaneously in separate PRs. ( https://guides.github.com/introduction/flow/ )

File: README.md (hunk 1 above)
Comment (krrohitch) [reply README.md:9]: Sir, this is an issue with Behance. It doesn't work on some connections

**Pull request merged**